tests/ttnn/unit_tests/operations/test_reduce_scatter_post_commit.py

```diff
@@ -106,6 +117,16 @@ def run_reduce_scatter_test(
         mismatch = mismatch or not eq
         if not eq:
             logger.error(f"output mismatch for tensor {i}")
+            if debug:
```

Comment:
mm, do we have a fully duplicated code in two places?

Reply:
will commonize